libraries/Microsoft.Bot.Schema/Teams/MeetingDetails.cs

```diff
@@ -62,7 +62,7 @@ public MeetingDetails()
         /// The meeting's scheduled start time, in UTC.
         /// </value>
         [JsonProperty(PropertyName = "scheduledStartTime")]
-        public DateTime ScheduledStartTime { get; set; }
+        public DateTime? ScheduledStartTime { get; set; }
```

Comment:
Isn't this a breaking change in that if I had code in my existing bot that was like this:

var hour = meetingDetails.ScheduledStartTime.Hour;

now that this is a nullable DateTime my code won't compile, and I'd need to change it to:

var hour = meetingDetails.ScheduledStartTime.Value.Hour;

as now that this is a nullable Date I have to use the .Value of it to get to the actual DateTime? And the same would be true for ScheduleEndTime as well. And while this may be a necessary change, based on SemVer it's a breaking change in that existing code will not continue to compile without changes if someone were to upgrade to the version this is shipped with.

Reply:
Hey @LeeParrishMSFT yes, this does break SemVer (not sure why our compat check missed it)

We had originally introduced strict SemVer binary compat specifically at the request of the Teams team, when they had their own package with direct dependency on the SDK. Now that the SDK has the Teams connector bits, I believe we can be a little less strict.

@clearab do you have any input on this?

Reply:
This is a very frustrating situation. It's true that fixing this would be breaking for anyone who is referencing that property _but this is a bug, the properties must allow nulls_ - and short of adding some smarts in the serailization what else can we do? other than introducing more properties which seems worse.

I would say lets take the change. But make sure to highlight this in the release notes.

The alternative being adding some smart code to our serialization that detects the "default" and makes that an undefined property on the wire. (I'm assuming we serialize "null" as undefined in the JSON not as null - but which ever the same reasoning applies.)